docs/specs/tripgo.swagger.yaml

```diff
@@ -2147,9 +2162,12 @@ definitions:
               type: string
             price:
               type: number
+            duration:
+              type: number
```

Comment:
Please add the unit. Is this hours or minutes?

Reply:
It is minutes. I have added a `description` field. Do you prefer a more descriptive field_name?

Reply:
Yes, maybe something like `maxDurationInMinutes`

Reply:
Renamed.